datafusion/sql/src/expr/mod.rs

```diff
@@ -468,11 +468,13 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
                 expr,
                 trim_where,
                 trim_what,
+                trim_characters,
```

Comment:
This is a pretty good example of the reason I prefer not to use `..` in patching patterns 

If we use `..` in the patterns and a new field is added, the compiler won't tell you on update where you may have to use it. 

I will make a follow on PR to ermove this `..` match

Reply:
https://github.com/apache/arrow-datafusion/pull/9531#